openmetadata-ui/src/main/resources/ui/src/components/Glossary/ImportGlossary/ImportGlossary.test.tsx

```diff
@@ -91,15 +91,23 @@ describe('Import Glossary', () => {
     const uploader = await screen.getByTestId('upload-file-widget');
     const uploadButton = await screen.getByTestId('upload-button');
     const stepper = await screen.getByTestId('stepper');
+    const cancelBtn = await screen.findByTestId('cancel-button');
 
     expect(breadcrumb).toBeInTheDocument();
     expect(title).toBeInTheDocument();
     expect(uploader).toBeInTheDocument();
     expect(uploadButton).toBeInTheDocument();
     expect(stepper).toBeInTheDocument();
+    expect(cancelBtn).toBeInTheDocument();
+
+    act(() => {
+      fireEvent.click(cancelBtn);
+    });
+
+    expect(mockPush).toHaveBeenCalledTimes(1);
```

Comment:
If test title says it should render all the component why are we adding assertion for Cancel button event here?